package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "commercetools-sunrise-theme",
-  "version": "0.68.0-SNAPSHOT",
```

Comment:
This will happen automatically when executing the release task. You should remove it from the code :)